tests/yast2_gui/yast2_security.pm

```diff
@@ -37,7 +39,14 @@ sub run {
     wait_still_screen 1;
     wait_screen_change { type_string "30" };
     wait_screen_change { send_key "alt-o" };
+    # on Aarch64, we need to refresh the x11 console to make desktop runner work
+    if (is_aarch64) {
+        reset_consoles;
+        wait_still_screen 10;
 
+        handle_logout;
```

Comment:
Could you confirm this logout/login is needed? Only needed between 1st and 2nd test, looks wired.

Reply:
> Could you confirm this logout/login is needed? Only needed between 1st and 2nd test, looks wired.

OK, I'd run w/o this logout/login, currently looks we just need it at 1st and 2nd launch. Thanks.

Reply:
@lemon-suse  Checked all ast2_gui_without_loginout[1-20] [passed](https://openqa.suse.de/tests/overview?build=hjluo&version=15-SP7&distri=sle&arch=aarch64). now remove the login/loginout.

Reply:
@lemon-suse  Can you please take a look again the comment to see if it's proper to explain the fix.